### ISSUE TITLE
Idea:  Quolling for Knowledge Checks and Student Response Questions for Formative Assessment in an Asynchronous Environment

### ISSUE THREAD

**Issue by rjf227** [elmsln/elmsln#622]
We have a lot of knowledge checks in courses and would like a tool that allows students to compare their answers with rest of the class.  

We would also like to to integrate Clicker/Poll Everywhere style questions in an asynchronous environment.  The graph display can also be used for formative assessment by instructors (to gauge how much knowledge is "sticking" and what concepts need reinforcement).

Would love to work a tool like this into eLMS:
Example of Polling/Knowledge question screenshot:  
<img width="619" alt="quolling-questions" src="https://cloud.githubusercontent.com/assets/15616843/13024843/78b64720-d1c9-11e5-9ed7-3703040eacbe.png">

Example of response display graph screenshot 

<img width="415" alt="quolling-display" src="https://cloud.githubusercontent.com/assets/15616843/13024848/85acb0ea-d1c9-11e5-875b-d283b10e51ad.png">

NOTE:  If you already voted, it says "YOU HAVE ALREADY VOTED" and users can't vote again.
Below you will find the link to download the code and the screen cases demo of the tool we saw demoed.

Demo:

http://laomedia.psu.edu/playlist.php?playlistID=31

Code:

 https://git.psu.edu/srt142/quolls

To download the code you will need to login with your PSU ID and password.

**Comment by btopro** [comment]
So we've already got capabilities like the baseline at least built into ICOR though we haven't documented how it works (or gotten it to filter out by section / running of the course, etc) so it's mostly in a dev state. It lets you pose simple questions w/ radio bubbles like above, then see the result as a barchart only (example)

![2016-02-12_20-49-00](https://cloud.githubusercontent.com/assets/329735/13024875/16d68410-d1ca-11e5-8e0d-1975ab9fdd46.png)

This is core drupal functionality that we've added a few minor levels of flexibility to.

As for the other question about clickers, I suggest checking out #299 as @heyMP / @juststormy piloted this in the Fall. We've got plans to port it to elmsln in the LQ / Live Question distribution but that hasn't happened yet.
